exercises/high-scores/high_scores_test.rb

```diff
@@ -0,0 +1,75 @@
+require 'minitest/autorun'
+require_relative 'high_scores'
+
+# Common test data version: 1.0.0 dd88548
```

Comment:
Maybe the Common test data version should be `0.0.0` until this gets merged into problem-specifications.